pytorch_toolkit/ote/ote/api.py

```diff
@@ -42,6 +34,21 @@ def train_args_parser(template_path):
 
     return parser
 
+def train_args_parser(template_path):
+    parser = argparse.ArgumentParser(parents=[compression_args_parser(template_path)],
+                                     formatter_class=argparse.RawDescriptionHelpFormatter,
+                                     add_help=False)
```

Comment:
will it show `--epochs` and `--base-learning-rate` in case of running script with `--help`?

Reply:
Yes. I've checked it.